### DISCUSSION TITLE
Just a test

### DISCUSSION THREAD

**Discussion by Tisfy** [LetMeFly666/LeetCode#6] [Q&A] [answered]
Hello, I'm just testing

**Comment by LetMeFly666** [comment]
So do I

**Comment by LetMeFly666** [comment]
> Hello, I'm just testing

So do I

**Comment by LetMeFly666** [comment, marked as answer]
It seems that only with a '[Q&A](https://github.com/LetMeFly666/LeetCode/discussions/categories/q-a)' level could get a 'Mark as answer'
